packages/vm/Cargo.toml

```diff
@@ -20,6 +20,8 @@ staking = ["cosmwasm-std/staking"]
 stargate = ["cosmwasm-std/stargate"]
 # Use cranelift backend instead of singlepass. This is required for development on Windows.
 cranelift = ["wasmer/cranelift"]
+# For heap profiling. Only used in "memory" example.
+dhat-heap = ["dep:dhat"]
```

Comment:
Can't we do the same with an optional dev-dependency? I'm a bit concerned this blows up people's Cargo.Lock even if unused.

Reply:
Unfortunately, there is no such thing as optional dev-dependencies.
It only gets added to their Cargo.lock if they enable the feature.